variables.tf

```
@@ -55,6 +55,12 @@ variable "truefoundry_db_database_name" {
 variable "truefoundry_db_ingress_security_group" {
   type        = string
   description = "SG allowed to connect to the database"
+  default     = ""
+
+  validation {
+    condition     = !var.truefoundry_db_enabled || var.truefoundry_db_ingress_security_group != ""
```

Condition should be more understandable
```suggestion
    condition     = var.truefoundry_db_enabled ? var.truefoundry_db_ingress_security_group != "" : true
```